src/nxlink.c

```diff
@@ -513,9 +513,10 @@ int main(int argc, char **argv) {
 		return -1;
 	}
 
+
 	char *msyscon = getenv("MSYSCON");
 
-	if (0 == strcmp(msyscon,"mintty.exe")) {
+	if (msyscon && 0 == strcmp(msyscon,"mintty.exe")) {
```

Comment:
Any reason why the 0 cmp is before strcmp and not after?

Reply:
it's just a style thing I tend to do to help flag typos with comparisons. i.e. if (var = 0) didn't used to get flagged by compilers but if(0 = var) would.